code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -404,5 +404,6 @@ UI STUFF
 
 /datum/controller/subsystem/tickets/proc/takeTicket(var/index)
 	if(assignStaffToTicket(usr.client, index))
-		message_staff("[span_text][usr.client] / ([usr]) has taken [ticket_name] number [index]</span>")
+		span_text = "<span class='adminchannel'>"
```

Comment:
This will break the other messages coming after these ones

Reply:
Nope, https://github.com/ParadiseSS13/Paradise/pull/11553 span closing is there.